doc/index.doc

```diff
@@ -122,12 +122,12 @@ The third part provides information for developers:
 \addindex license
 \addindex GPL
 
-Copyright &copy; 1997-\showdate "%Y" by <a href="mailto:doxygen@gmail.com">Dimitri van Heesch</a>.<p>
+Copyright &copy; 1997-\showdate "%Y" &nbsp;by <a href="mailto:doxygen@gmail.com">Dimitri van Heesch</a>.<p>
```

Comment:
I understand this will force a space, but I think it would be more intuitive if the space after `"\showdate %y"` is preserved. What do you think?

Reply:
That would have been nice but I think that again would limit probably other possibilities where it would be possible to join things directly with the (in this case year), like the result of other commands as at that moment the string would not a proper forma string. In a bit an artificial example:
```
\showdate "%Y" \showdate "%Y"
```
would result now in `20232023` but when the space is retained it would be `2023 2023`.
when using
```
\showdate "%Y"\showdate "%Y"
```
we get:
```
warning: invalid <date_time> argument for command '\showdate'
```

(I know that in this special case I could use "%Y%Y")